packages/framework-provider-local/src/services/read-model-registry.ts

```diff
@@ -0,0 +1,40 @@
+import { ReadModelEnvelope, UUID } from '@boostercloud/framework-types'
+import * as DataStore from 'nedb'
+import { readModelsDatabase } from '../paths'
+
+export class ReadModelRegistry {
+  public readonly readModels: DataStore<ReadModelEnvelope> = new DataStore(readModelsDatabase)
+  constructor() {
+    this.readModels.loadDatabase()
+  }
+
+  public async query(query: object): Promise<Array<ReadModelEnvelope>> {
+    const queryPromise = new Promise((resolve, reject) =>
+      this.readModels.find(query).exec((err, docs) => {
+        if (err) reject(err)
+        else resolve(docs)
+      })
+    )
+
+    return (await queryPromise) as Array<ReadModelEnvelope>
+  }
+
+  public async store(readModel: ReadModelEnvelope): Promise<void> {
+    return new Promise((resolve, reject) => {
+      this.readModels.update({ id: readModel.value.id }, readModel, { upsert: true }, (err) => {
+        err ? reject(err) : resolve()
+      })
+    })
+  }
+
+  public async deleteById(id: UUID): Promise<number> {
+    const deletePromise = new Promise((resolve, reject) =>
+      this.readModels.remove({ value: { id } }, { multi: true }, (err, numRemoved: number) => {
+        if (err) reject(err)
+        else resolve(numRemoved)
+      })
+    )
+
+    return (await deletePromise) as number
```

Comment:
Same here, I think you don't need the async/await

Reply:
Done in https://github.com/boostercloud/booster/pull/473/commits/3c8b3a4c2f22227b10e16c3689c8c1f4c03e051d